homeassistant/components/dlink/manifest.json

```diff
@@ -4,6 +4,7 @@
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/dlink",
   "requirements": ["pyW215==0.7.0"],
+  "dhcp": [{ "hostname": "dsp-w215" }],
```

Comment:
Is that the exact hostname or does this need a wildcard?

Reply:
Exact hostname. D-Link has many blocks of mac addresses. 3 of them are used between the 5 I have so we'll leave that part out.

Reply:
debug logs confirm a match